lib/timescaledb/acts_as_hypertable/core.rb

```diff
@@ -35,7 +35,7 @@ def define_association_scopes
             CompressionSettings.where(hypertable_name: table_name)
           end
 
-          scope :continuous_aggregates, -> do
+          scope :caggs, -> do
```

Comment:
I got some mad issue with this name, still not sure how to handle the naming clash. Just renaming this one as seems less prioritized.